### PR TITLE
add check of date

### PR DESCRIPTION
**Pull request by maelle**
cf #20

### CODE REVIEW

File: R/cranlogs.R [hunk 1]
```diff
@@ -62,6 +62,14 @@ cran_downloads <- function(packages = NULL,
   if (!missing(when)) {
     interval <- match.arg(when)
   } else {
+    if (from != "last-day"){
+      check_date(from)
+    }
+    
+    if (to != "last-day"){
+      check_date(to)
```
Comment (gaborcsardi) [inline R/cranlogs.R]: please add space before `{`, above as well

Comment (maelle) [reply R/cranlogs.R]: done!

File: R/utils.R [hunk 2]
```diff
@@ -0,0 +1,6 @@
+check_date <- function(string){
+  if(is.na(as.Date(string, format = "%Y-%m-%d"))){
```
[the remark this answers is not in the document]
Comment (maelle) [reply R/utils.R]: :heavy_check_mark:

**Comment by maelle** [comment]
Re-styled! :nail_care:

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**